src/module/config.ts

```diff
@@ -95,9 +95,9 @@ export const OSE = {
     shield: "OSE.armor.shield",
   },
   apply_damage_options: {
-    selected : "selected",
-    targeted : "targeted",
-    originalTarget : "originalTarget",
```

Comment:
This will break runtime, as the values in this config are used by `helper-chats.ts` for comparison with the setting(so they need to be equal to the setting value, not the text)

see
https://github.com/vttred/ose/blob/main/src/module/helpers-chat.ts#L16

Reply:
You're absolutely right, thanks!